web/templates/user_profile_modal.hbs

```diff
@@ -90,6 +90,13 @@
                     </div>
 
                     <div class="tabcontent" id="user-profile-streams-tab">
+                        <div class="alert stream_list_info"></div>
+                        {{#if show_user_subscribe_widget}}
+                            <div class="header-section">
+                                <h3 class="subscribe-widget-header">{{t 'Subscribe'}} {{full_name}} {{t 'to streans'}}</h3>
```

Comment:
Internationalization doesn't work that way -- you need the whole sentence in the string with a variable in it.
https://zulip.readthedocs.io/en/latest/translating/internationalization.html

(Different languages have different word orders)

Reply:
Thanks for pointing, fixed it.